ludwig/features/image_feature.py

```diff
@@ -294,16 +313,19 @@ def add_feature_data(
             # standard code anyway.
             if backend.supports_multiprocessing and (
                     num_processes > 1 or num_images > 1):
-                all_file_paths = [get_abs_path(src_path, file_path)
-                                  for file_path in input_df[feature[NAME]]]
+                all_img_stores = [get_abs_path(src_path, img_store)
+                                  if isinstance(img_store, str) else img_store
+                                  for img_store in input_df[feature[NAME]]]
```

Comment:
This should most likely be `in input_df[feature[COLUMN]]` instead. It could have been a leftover error from when we introduced the COLUMN concept

Reply:
changed to `input_df[feature[COLUMN]]`